### PR TITLE
Update lambda argument parsing, the JSON is pre-parsed for us

### PR DESCRIPTION
**Pull request by zachmargolis**

### CODE REVIEW

File: source/proof_address/lib/proof_address.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@
 module IdentityIdpFunctions
   class ProofAddress
     def self.handle(event:, context:, &callback_block)
```
Comment (zachmargolis) [inline source/proof_address/lib/proof_address.rb:11]: this affects the calling code in the IDP so update that when I bump this in the IDP

File: source/proof_address/lib/proof_address.rb [hunk 2]
```diff
@@ -9,7 +9,7 @@
...
-      params = JSON.parse(event['body'], symbolize_names: true)
+      params = event.transform_keys(&:to_sym)
```
Comment (mitchellhenke) [inline source/proof_address/lib/proof_address.rb:12]: Do `symbolize_names` and `transform_keys` differ in how they handle nested maps?

**Review by mitchellhenke — APPROVED** [review]
One question 👍🏼

File: source/proof_address/lib/proof_address.rb (hunk 2 above)
Comment (zachmargolis) [reply source/proof_address/lib/proof_address.rb:12]: oh yikes good catch, it does not.

```
{ 'a' => { 'b' => 1 } }.transform_keys(&:to_sym)
# => {:a=>{"b"=>1}}
```

Gonna try plan B

Comment (zachmargolis) [reply source/proof_address/lib/proof_address.rb]: Plan B: 818b9411ebe3ee0f0196fb41d92a61b8ce040485

**Pull request merged**